src/DomainEventPropagation/DomainEventWrapper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.Json;
+
+namespace Workleap.DomainEventPropagation;
+
+internal class DomainEventWrapper : IDomainEvent
```

Comment:
Can we add `JsonPropertyName` attributes with single letter JSON property name in order to reduce the size of the generated JSON? I suggest:

DomainEventType: `t`
DomainEventJson: `j`
ExtensionAttributes: `e`

Reply:
Yeah good point, will add it!